pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/AvroSchema.java

```diff
@@ -118,6 +118,8 @@ public static void addLogicalTypeConversions(ReflectData reflectData, boolean js
         reflectData.addLogicalTypeConversion(new TimeConversions.DateConversion());
         reflectData.addLogicalTypeConversion(new TimeConversions.TimeMillisConversion());
         reflectData.addLogicalTypeConversion(new TimeConversions.TimeMicrosConversion());
+        reflectData.addLogicalTypeConversion(new TimeConversions.LocalTimestampMicrosConversion());
+        reflectData.addLogicalTypeConversion(new TimeConversions.LocalTimestampMillisConversion());
```

Comment:
why weren't these put under the if statement `if (jsr310ConversionEnabled) {` ?

Reply:
Because there is no Joda time conversion of `local-timestamp-millis` and` local-timestamp-micros'.